views/dashboard.handlebars

```diff
@@ -4,7 +4,7 @@
     <h2> Welcome {{userData.name}}</h2>
     <div id="todayDeck">
       <h3>Today's Deck:</h3>
-      <button id="todaysDeck" onclick="window.location = '/review/all'">Review all Cards</button>
+      <button class="alldeckbutton" id="todaysDeck"><a href="/review/all">Review all Cards</a></button>
```

Comment:
Adding the `alldeckbutton` class is ok, but I'd also like to keep the `onclick="window.location = 'review/all'"` and get rid of the nested `a` tag.

Reply:
My apologies! I did not actually mean to remove the onclick function! I will add it back!